client/src/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<head>
+  <meta charset="utf-8">
+  <title>My App</title>
+
+</head>
+
+<div id="app" ></div>
+
+<script
+        src="https://code.jquery.com/jquery-3.2.1.js"
```

Comment:
You are using react, you don't need jquery at all.
Jquery was good in 2010 :)
http://youmightnotneedjquery.com/